biscuit-auth/src/format/schema.proto

```diff
@@ -178,3 +181,39 @@ message ThirdPartyBlockContents {
   required bytes payload = 1;
   required ExternalSignature externalSignature = 2;
 }
+
+message AuthorizerSnapshot {
```

Comment:
i think this should be named `AuthorizerWorld`, since a complete snapshot will require tracking more information (at least the iteration count and elapsed time, maybe execution limits and a flag tracking whether evaluation has started or not)

Reply:
fixed in [2f8aca5](https://github.com/biscuit-auth/biscuit-rust/pull/127/commits/2f8aca5358966c0f43ec761ebd6279beebfc0bcd). I'm not sure about the u64 to store the durations though